server/base/src/main/java/org/apache/accumulo/server/metadata/TabletMutatorBase.java

```diff
@@ -151,6 +166,20 @@ private String getLocationFamily(LocationType type) {
     return this;
   }
 
+  @Override
+  public Ample.TabletMutator putBulkFile(Ample.FileMeta bulkref, long tid) {
+    Preconditions.checkState(updatesEnabled, "Cannot make updates after calling mutate.");
+    Value tidBytes = new Value(Long.toString(tid).getBytes());
+    mutation.put(TabletsSection.BulkFileColumnFamily.NAME, bulkref.meta(), tidBytes);
+    return this;
+  }
+
+  // not sure what to do here yet
+  @Override
+  public Ample.TabletMutator deleteBulkFile(Ample.FileMeta bulkref) {
+    throw new UnsupportedOperationException();
```

Comment:
Probably the following

```java
Preconditions.checkState(updatesEnabled, "Cannot make updates after calling mutate.");
mutation.putDelete(TabletsSection.BulkFileColumnFamily.NAME, bulkref.meta());
```

Reply:
Added this although nothing out there uses it at this time.